pandas/plotting/matplotlib/main.py

```diff
@@ -0,0 +1,1374 @@
+import re
```

Comment:
Maybe "core" instead of "main"? I typically associate main with `__main__` / running from the command line.